src/frequenz/channels/_broadcast.py

```diff
@@ -161,15 +159,15 @@ def new_receiver(
 
         Args:
             name: A name to identify the receiver in the logs.
-            maxsize: Size of the receiver's buffer.
+            limit: Size of the receiver's buffer in number of messages.
```

Comment:
Maybe this is better?

```suggestion
            limit: Number of messages the receiver can hold in its buffer.

Reply:
Updated with the suggestion.